spec/template.rb

```diff
@@ -1,5 +1,6 @@
 generate 'rspec:install'
 
 directory 'spec', force: true
```

Comment:
Can we pass the option `exclude_pattern: /pattern/` to directory action?
[That action supports](https://www.rubydoc.info/github/erikhuda/thor/master/Thor/Actions:directory) to exclude files with pattern

Reply:
That works @hoangmirs , thanks for the suggestion, fixed it in https://github.com/nimblehq/rails-templates/pull/212/commits/1e4688bc43ae0a0803139306b6c06dd51ea5b64c